src/collections/_documentation/performance-monitoring/performance/event-detail.md

```diff
@@ -20,20 +20,30 @@ Currently, only root transactions are searchable. Any span data that inherits fr
 
 ## Span View
 
+The span view is a split view where the left-hand side shows the transaction’s span tree, and the right-hand side represents each span as a colored rectangle. Within the tree view, Sentry identifies spans by their `op` and `description` values. If a span doesn’t have a description, Sentry uses the span’s id as a fallback. The first span listed is always the transaction’s root span, from which all other spans in the transaction descend.
+
 [{% asset performance/span-details.png alt="Span detail view shows the span id, trace id, parent span id, and other data such as tags." %}]({% asset performance/span-details.png @path %})
 
+To find these views, you can either go through the [Transaction Summary](/performance-monitoring/performance/transaction-summary) or [Query Builder](/performance-monitoring/discover-queries/query-builder/). Event IDs will be linked to open the corresponding Event Detail.
```

Comment:
@doralchan I updated lines 29 and 33 to align with this paragraph. Will you double check the accuracy?